src/dbcp/metadata/private_data_warehouse.py

```diff
@@ -173,3 +178,104 @@
     Column("raw_storage_durations", String),
     schema=schema,
 )
+
+######################################
+# Interconnection.fyi Project Queues #
+######################################
+fyi_projects = Table(
+    "fyi_projects",
+    metadata,
+    Column("project_id", String, primary_key=True, autoincrement=False),
+    Column("project_type", String),
+    Column("power_market", String, nullable=False),
+    Column("transmission_owner", String),
+    Column("developer", String),
+    Column("developer_raw", String),
+    Column("canonical_transmission_owners", String),
+    Column("interconnection_status_fyi", String),
+    Column("interconnection_status_raw", String),
+    Column("queue_id", String),
+    Column("project_name", String),
+    Column("actual_completion_date", DateTime, nullable=True),
+    Column("proposed_completion_date", DateTime, nullable=True),
+    Column("withdrawn_date", DateTime, nullable=True),
+    Column("queue_date", DateTime),
+    Column("queue_year", Integer),
+    Column("county_state_pairs", String),
+    Column("point_of_interconnection", String),
+    Column("canonical_generation_types", String),
+    Column("interconnection_service_type", String),
+    Column("interconnection_date", DateTime),
+    Column("interconnection_date_raw", String),
+    Column("queue_status", String, nullable=False),
+    Column("summer_capacity_mw", Float),
+    Column("winter_capacity_mw", Float),
+    Column("queue_status", String, nullable=False),
+    Column("current_phase_or_stage_raw", String),
+    Column("project_spv", String),
+    Column("utility", String),
+    Column("iso", String),
+    Column("cluster", String),
+    Column("general_comments", String),
+    Column("interconnection_voltage_kv", String),
+    Column("latitude", Float),
+    Column("longitude", Float),
+    Column("schedule_next_event_date", DateTime),
+    Column("schedule_next_event_name", String),
+    Column("most_recent_study_date", DateTime),
+    Column("most_recent_study_url", String),
+    Column("most_recent_allocated_network_upgrade_cost", Float),
+    Column("is_actionable", Boolean),
+    Column("is_nearly_certain", Boolean),
+    schema=schema,
+)
+
+fyi_locations = Table(
+    "fyi_locations",
+    metadata,
+    Column(
+        "project_id",
+        String,
+        ForeignKey("private_data_warehouse.fyi_projects.project_id"),
+    ),
+    Column("raw_county_name", String),
+    Column("raw_state_name", String),
+    Column(
+        "state_id_fips",
+        String,
+        nullable=True,
+    ),
+    Column(
+        "county_id_fips",
+        String,
+        nullable=True,
+    ),
+    ForeignKeyConstraint(
+        ["county_id_fips"],
+        ["data_warehouse.county_fips.county_id_fips"],
+        link_to_name=True,
+    ),
+    ForeignKeyConstraint(
+        ["state_id_fips"],
+        ["data_warehouse.state_fips.state_id_fips"],
+        link_to_name=True,
+    ),
```

Comment:
@zschira sqlalchemy isn't happy about this fk constraint to a table in a different schema. It can't find the county_fips and state_fips tables. Need to fix.